LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs

```diff
@@ -297,5 +297,74 @@ public void SplitTenderExample_LevelUp_Then_Cash_With_Adjusted_Exemption_And_Adj
                 $"Expected: {expectedCompletedOrderValues}" + Environment.NewLine +
                 $"Actual: {completedOrderValues}");
         }
+        
+        /// <summary>
+        /// In this example, cash is applied first.  LevelUp is paid for second, and the remaining balance can be
+        /// covered in full by the LevelUp discount credit available.
+        /// </summary>
+        /// <remarks>
+        /// In this example, the second payment will still be responsible for some tax.
+        /// </remarks>
+        [TestMethod]
+        public void SplitTenderExample_LevelUp_Second_Discount_Remaining_Balance()
+        {
+            // $22.00 is owed, $2.00 of that is tax. $0.00 of that is tobacco/alcohol, and the customer wants to pay
+            // $12.00 towards the check all of which is discountable.
+            int originalSubtotal = 2000;
+            int originalTotal = 2200;
+
+            // Apply a non-LevelUp tender
+            int cashPayment = 1000;
+
+            int currentSubtotal = originalSubtotal - cashPayment;
+            int currentTotal = originalTotal - cashPayment;
+            int currentTax = currentTotal - currentSubtotal;
+
+            int exemptionAmount = 0;
+            int spendAmount = 1200;
+
+            // Create proposed order: expected values
+            AdjustedCheckValues expectedProposedOrderValues =
+                new AdjustedCheckValues(
+                    spendAmount: 1200,
+                    taxAmount: 200,
+                    exemptionAmount: 0);
+
+            AdjustedCheckValues proposedOrderValues = ProposedOrderCalculator.CalculateCreateProposedOrderValues(
+                currentTotal,
+                currentTax,
+                exemptionAmount,
+                spendAmount);
+
+            proposedOrderValues.ShouldBeEquivalentTo(expectedProposedOrderValues,
+                $"Expected: {expectedProposedOrderValues}" + Environment.NewLine +
+                $"Actual: {proposedOrderValues}");
+
+            // available discount amount $10.00
+            int availableDiscountAmount = 1000;
+
+            // POS applies $10.00 pretax discount to check and updates subtotal and tax
+            currentTax = (originalSubtotal - availableDiscountAmount) / TAX_RATE;
```

Comment:
Why use `TAX_RATE` here to calculate tax, but not above?  I realize the end result is the same, but the style difference is odd and may lead one to wonder if there's a problem.